### PR TITLE
Themes | Business Dashboard - TableListItem - add theme

### PR DESCRIPTION
**Pull request by netanel89**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: packages/wix-style-react/src/Themes/businessDashboard/components/TableListItem.st.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  -st-from: '../foundations/colors.st.css';
+  -st-named: B50;
+}
+
+.root {
+  -st-extends: TableListItem;
+}
+
+.root:verticalPadding(small) {
+  padding: 0 0 12px 0;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/TableListItem.st.css:16]: We have SP3 in our spacing.st.css

File: packages/wix-style-react/src/Themes/businessDashboard/components/TableListItem.st.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.root:verticalPadding(small):not(:first-child) {
+  padding-top: 12px;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/TableListItem.st.css:20]: same

File: packages/wix-style-react/src/Themes/businessDashboard/docs/components/TableListItem.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        options={[
+          { value: 'Personal Finance' },
+          { value: '7 posts' },
+          { value: 'Last update on 27 April 2020' },
+        ]}
+        onClick={() => { }}
+      />
+    </Layout>
+  </ThemeProvider>
+`;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/docs/components/TableListItem.js:15]: consider importing columnAlignmentExample from TableListItem/docs/examples.js

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6827/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6827

File: packages/wix-style-react/src/Themes/businessDashboard/docs/components/TableListItem.js (hunk 3 above)
Comment (netanel89) [reply packages/wix-style-react/src/Themes/businessDashboard/docs/components/TableListItem.js]: It does not contain onClick event

**Review by adicicurel — APPROVED** [review]

**Comment by netanel89** [comment]
@mykas Can you please review / merge ?

**Pull request merged**